assets/xml/objects/object_vm.xml

```diff
@@ -1,30 +1,41 @@
-        <Texture Name="object_vm_Tex_002E40" OutName="tex_002E40" Format="rgba16" Width="16" Height="16" Offset="0x2E40" />
-        <Texture Name="object_vm_Tex_003040" OutName="tex_003040" Format="rgba16" Width="32" Height="16" Offset="0x3040" />
-        <Texture Name="object_vm_Tex_003440" OutName="tex_003440" Format="rgba16" Width="16" Height="16" Offset="0x3440" />
-        <Texture Name="object_vm_Tex_003640" OutName="tex_003640" Format="rgba16" Width="8" Height="8" Offset="0x3640" />
-        <Texture Name="object_vm_Tex_0036C0" OutName="tex_0036C0" Format="rgba16" Width="32" Height="32" Offset="0x36C0" />
-        <Limb Name="object_vm_Standardlimb_003EC0" Type="Standard" EnumName="OBJECT_VM_LIMB_01" Offset="0x3EC0" />
-        <Limb Name="object_vm_Standardlimb_003ECC" Type="Standard" EnumName="OBJECT_VM_LIMB_02" Offset="0x3ECC" />
-        <Limb Name="object_vm_Standardlimb_003ED8" Type="Standard" EnumName="OBJECT_VM_LIMB_03" Offset="0x3ED8" />
-        <Limb Name="object_vm_Standardlimb_003EE4" Type="Standard" EnumName="OBJECT_VM_LIMB_04" Offset="0x3EE4" />
-        <Limb Name="object_vm_Standardlimb_003EF0" Type="Standard" EnumName="OBJECT_VM_LIMB_05" Offset="0x3EF0" />
-        <Limb Name="object_vm_Standardlimb_003EFC" Type="Standard" EnumName="OBJECT_VM_LIMB_06" Offset="0x3EFC" />
-        <Limb Name="object_vm_Standardlimb_003F08" Type="Standard" EnumName="OBJECT_VM_LIMB_07" Offset="0x3F08" />
-        <Limb Name="object_vm_Standardlimb_003F14" Type="Standard" EnumName="OBJECT_VM_LIMB_08" Offset="0x3F14" />
-        <Limb Name="object_vm_Standardlimb_003F20" Type="Standard" EnumName="OBJECT_VM_LIMB_09" Offset="0x3F20" />
-        <Limb Name="object_vm_Standardlimb_003F2C" Type="Standard" EnumName="OBJECT_VM_LIMB_0A" Offset="0x3F2C" />
-        <Skeleton Name="object_vm_Skel_003F60" Type="Normal" LimbType="Standard" LimbNone="OBJECT_VM_LIMB_NONE" LimbMax="OBJECT_VM_LIMB_MAX" EnumName="ObjectVmLimb" Offset="0x3F60" />
+        <!-- Beamos Animations -->
+        <Animation Name="gBeamosAnim" Offset="0x68" />
+        
+        <!-- Beamos Limb DisplayLists -->
+        <DList Name="gBeamosBodyDL" Offset="0x1E00" />
+        <DList Name="gBeamosTopEyelidDL" Offset="0x22B0" />
+        <DList Name="gBeamosBottomEyelidDL" Offset="0x2460" />
+        <DList Name="gBeamosEyeDL" Offset="0x2610" />
+
+        <!-- Beamos Laser DisplayList -->
+        <DList Name="gBeamosLaserDL" Offset="0x2728" />
+
+        <!-- Beamos DisplayList Textures -->
+        <Texture Name="gBeamosBodyGradientTex" OutName="body_gradient" Format="rgba16" Width="8" Height="8" Offset="0x27C0" />
+        <Texture Name="gBeamosGrayMetalTex" OutName="gray_metal" Format="i4" Width="16" Height="32" Offset="0x2840" />
+        <Texture Name="gBeamosBoltAndMetalTex" OutName="bolt_and_metal" Format="rgba16" Width="16" Height="32" Offset="0x2940" />
+        <Texture Name="gBeamosTex_2D40" OutName="tex_2D40" Format="rgba16" Width="16" Height="8" Offset="0x2D40" />
```

Comment:
```suggestion
        <Texture Name="gBeamosInnerEyeLidTex" OutName="beamos_inner_eye_lid" Format="rgba16" Width="16" Height="8" Offset="0x2D40" />
```
Looked at it in z64utils as well as Texture64 and this would be my naming suggestion.